packages/core/src/embeddings/OpenAIEmbedding.ts

```diff
@@ -109,6 +123,16 @@ export class OpenAIEmbedding extends BaseEmbedding {
    * @param options
    */
   private async getOpenAIEmbedding(input: string[]): Promise<number[][]> {
```

Comment:
this is not a good solution.

This will break the return type of `getTextEmbedding('large chunk')`

Reply:
https://github.com/run-llama/LlamaIndexTS/pull/843#issue-2299093676

Reply:
doesn't `getTextEmbedding('large chunk')` - it just ensures that `getTextEmbedding('large chunk')` is not throwing an exception